omnibus/config/software/updater.rb

```diff
@@ -42,8 +42,6 @@
     mkdir "/etc/init"
     mkdir "/var/log/datadog"
 
-    move 'bin/agent/dist/datadog.yaml', '/etc/datadog-agent/datadog.yaml.example'
```

Comment:
I will add back examples in further work, those files [currently don't exist in bin/agent/dist](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/393834844)